torchvision/io/__init__.py

```diff
@@ -32,6 +32,10 @@
     write_video,
 )
 
+from .gpu_decoder import (
```

Comment:
Shouldn't this be somehow guarded (if it exists, if there is a cuda device, etc...)

Reply:
That's a good point. We could perhaps do that in the GPUDecoder class.